src/pages/FindYourCommunity/YourLocalArea/FindCharity/FindCharity.tsx

```diff
@@ -88,6 +89,9 @@ const FindCharity: FC = () => {
           columns={columns}
           scroll={{ x: 'max-content' }}
           rowKey="id"
+          locale={{
```

Comment:
Can we repeat this for the donate and excess pages? there are charity tables on those too.

Reply:
thanks, I've now done this